UnityProject/Assets/Scripts/UI/PreRound/GUI_PreRoundWindow.cs

```diff
@@ -77,9 +78,9 @@ private void OnDisable()
 
 	private void Update()
 	{
-		// TODO: remove once admin system is in
-		if (Input.GetKeyDown(KeyCode.F7) && !BuildPreferences.isForRelease)
+		if (Input.GetKeyDown(KeyCode.F7))
```

Comment:
does this still work if you're in the editor

Reply:
Yup

Reply:
It worked in builds too, so the build preferences.isForRelease didnt do anything anyway?